MapboxVision/src/main/java/com/mapbox/vision/video/videosource/camera/Camera2VideoSourceImpl.kt

```diff
@@ -219,9 +223,12 @@ class Camera2VideoSourceImpl(
                     .get(CameraCharacteristics.LENS_INFO_AVAILABLE_FOCAL_LENGTHS)!!
                     .first()
 
-                val physicalSensorSize = characteristics.get(CameraCharacteristics.SENSOR_INFO_PHYSICAL_SIZE)!!
-                val focalInPixelsX: Float = focalLength * previewSize.imageWidth / physicalSensorSize.width
-                val focalInPixelsY: Float = focalLength * previewSize.imageHeight / physicalSensorSize.height
+                val physicalSensorSize =
+                    characteristics.get(CameraCharacteristics.SENSOR_INFO_PHYSICAL_SIZE)!!
+                val focalInPixelsX: Float =
+                    focalLength * previewSize.imageWidth / physicalSensorSize.width
```

Comment:
`physicalSensorSize.width==0` ?

Reply:
This is the physical size of the sensor pixel. This key is available on all devices